src/arch.rs

```diff
@@ -820,6 +820,131 @@ registers!(X86_64, {
     K7 = (125, "k7"),
 });
 
+/// PowerPC 64bit
+///
+/// See [64-bit ELF ABI Specification for OpenPOWER Architecture](https://openpowerfoundation.org/specifications/64bitelfabi/).
+#[derive(Debug, Clone, Copy)]
+pub struct PowerPc64;
+
+registers!(PowerPc64, {
+    R0 = (0, "R0"),
```

Comment:
For the string name, is it more common to use lowercase or uppercase? The linked document uses lowercase.

Reply:
Lowercase names would be more common, I'll change to those.